src/app/app-design-tool/components/jobs-list/components/job/job.component.html

```diff
@@ -0,0 +1,8 @@
+<mat-list-item>
```

Comment:
This is a very nice separation! don't forget to add the license here and the other files.